frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/common-template-validations/selectors.ts

```diff
@@ -0,0 +1,114 @@
+import { iGetIn } from '../../../../../utils/immutable';
+import {
+  TEMPLATE_TYPE_LABEL,
+  TEMPLATE_WORKLOAD_LABEL,
+  TEMPLATE_OS_LABEL,
+  TEMPLATE_FLAVOR_LABEL,
+} from '../../../../../constants/vm';
+import {
+  CommonTemplatesValidation,
+  IntegerValidtionType,
+  CommonTemplatesValidationAttr,
+  IntegerValidationObj,
+  IntegerBetween,
+} from './validation-types';
+
+export const getRelevantTemplates = (commonTemplates, os, workloadProfile, flavor) => {
```

Comment:
can we move this to vm-template selectors?

Reply:
we can. no problem.